client/src/views/devices/GreetDeviceModal.vue

```diff
@@ -283,30 +308,28 @@ interface GreetDeviceTitle {
   subtitle?: Translatable;
 }
 
-function getTitleAndSubtitle(): GreetDeviceTitle {
-  if (pageStep.value === GreetDeviceStep.WaitForGuest) {
-    return { title: 'DevicesPage.greet.titles.waitForGuest' };
-  } else if (pageStep.value === GreetDeviceStep.ProvideHostSasCode) {
-    return {
+function getStep(): Array<GreetDeviceTitle> {
```

Comment:
Since the function just returns, might as well do

`const steps = ref([
  {title: ...},
  {title: ...},
  ...
])`

Way easier to access.